caddytls/setup_test.go

```diff
@@ -358,6 +358,58 @@ func TestSetupParseWithOneTLSProtocol(t *testing.T) {
 	}
 }
 
+func TestSetupParseCertificateTransparencyDefault(t *testing.T) {
+	// This isn't must_staple specific, we just need something in that block.
```

Comment:
Haha, I guess this is kind of a funny quirk of testing this directive. Sorry about that.